OpenRA.Mods.Common/Traits/Air/Aircraft.cs

```diff
@@ -158,7 +158,7 @@ IEnumerable<object> IActorPreviewInitInfo.ActorPreviewInits(ActorInfo ai, ActorP
 		bool IOccupySpaceInfo.SharesCell { get { return false; } }
 
 		// Used to determine if an aircraft can spawn landed
-		public bool CanEnterCell(World world, Actor self, CPos cell, Actor ignoreActor = null, bool checkTransientActors = true)
+		public bool CanEnterCell(World world, Actor self, CPos cell, Actor ignoreActor = null, bool checkTransientActors = true, SubCell subCell = SubCell.FullCell)
```

Comment:
Shouldn't we be using `subCell` in the `GetActorsAt` check below?

Reply:
Aircraft never use subcells (see the property just above), so I thought this would be irrelevant (or possibly slower since the overload with subcells does more things).

Reply:
This is fair enough, but can you please add comments so it is clear that this was an intentional choice rather than oversight?